internal/jobcontainers/jobcontainer.go

```diff
@@ -237,8 +307,19 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		envs = append(envs, k+"="+expanded)
 	}
 	env = append(env, envs...)
-
-	env = append(env, sandboxMountPointEnvVar+"="+c.sandboxMount)
+	env = append(env, sandboxMountPointEnvVar+"="+c.rootfsLocation)
+
+	// Add the rootfs location to PATH so you can run things from the root of the image.
+	for idx, envVar := range env {
+		ev := strings.TrimSpace(envVar)
+		if len(ev) >= 5 && strings.ToLower(ev[:5]) == "path=" {
```

Comment:
`strings.HasPrefix(strings.ToLower(ev), "path=")`

Reply:
Yea that'd be a bit easier..

Reply:
Changed